src/MuTalk-Model/MutantEvaluation.class.st

```diff
@@ -85,3 +85,18 @@ MutantEvaluation >> value [
 		results: testResults
 		producedBy: self
 ]
+
+{ #category : 'evaluation' }
+MutantEvaluation >> valueWithoutStoppingOnError [
+
+	| testResults |
+	self initializeCoverageResultIfNil.
+
+	mutation testCaseReferences:
+		(strategy testCasesToEvaluate: mutation for: self).
+	testResults := mutation runMutantWithoutStoppingOnError.
+	^ MutantEvaluationResult
+		  for: mutation
+		  results: testResults
+		  producedBy: self
+]
```

Comment:
It bothers me a bit the repeated code between this and the old method.
Why not passing the boolean as parameter here, and just make the `if` on the line that *really changes*? :)